spec/overcommit/hook/pre_commit/rake_target_spec.rb

```diff
@@ -5,12 +5,12 @@
   let(:context) { double('context') }
   subject { described_class.new(config, context) }
 
-  # context 'without targets parameters' do
-  #   let(:result) { double('result') }
-  #   it 'raises' do
-  #     expect { subject }.to raise_error(RuntimeError)
-  #   end
-  # end
+  context 'without targets parameters' do
+    let(:result) { double('result') }
+    it 'raises' do
+      expect { subject }.to raise_error(RuntimeError, /RakeTarget: targets parameter is empty.*/)
```

Comment:
You need to call `subject.run` here in order to get the error thrown. This is done automatically in `spec/support/matchers/hook.rb` when you call `fail_hook` or `pass`. But since you're not using those helpers here you need to be more explicit.